tensorflow/compiler/xla/service/gpu/launch_dimensions.cc

```diff
@@ -122,27 +129,27 @@ StatusOr<LaunchDimensions> CalculateLaunchDimensions(
     }
   }
 
-  int64_t block_count = CeilOfRatio(num_elements, threads_per_block);
+  int64_t block_count = CeilOfRatio(num_elements,
+                                  threads_per_block * threads_per_block_y);
```

Comment:
That I do not follow. I thought `threads_per_block` already includes `threads_per_block_y` in itself. If not, shouldn't it be called `threads_per_block_x` or similar?